onnxruntime/python/session.py

```diff
@@ -194,18 +194,31 @@ class IOBinding:
     def __init__(self, session):
```

Comment:
Can we add docstrings to this class and its public methods? See InferenceSession.__init__ for an example.

Reply:
Done.